src/tests/ftest/recovery/ms_membership.py

```diff
@@ -146,7 +147,7 @@ def test_enable_disable_admin_excluded(self):
         if not excluded_found:
             errors.append("Rank 1 state is not Excluded! {}".format(rank_1_state))
 
-        # 7. Start rank 1 for cleanup.
-        dmg_command.system_start(ranks="1")
+        # 7. Start both of the servers for cleanup.
+        dmg_command.system_start()
```

Comment:
+1 that we should update the expected states when excluding.  I think it's also about setting a good pattern in case other tests need to do the same thing.

Reply:
Done. I added `system_exclude()` and `system_clear_exclude()` in `DaosServerManager`.